src/net/sf/memoranda/ui/DailyItemsPanel.java

```diff
@@ -90,7 +90,7 @@
     FlowLayout flowLayout1 = new FlowLayout();
     JButton taskB = new JButton();
     JPanel mainTabsPanel = new JPanel();
-    NotesControlPanel notesControlPane = new NotesControlPanel();
+    PSPControlPanel notesControlPane = new PSPControlPanel();
```

Comment:
Since we found out the `editorPanel` was actually the content pane we were targeting, I looked into what this object was controlling, and it was actually in charge of the panel under the calendar. Since we don't need that anymore, you can actually remove every instance of `notesControlPane` from the program (there are about 6; in this class and the `AppFrame` class).